backbone/src/main/res/layout/rsb_image_capture_step_layout.xml

```diff
@@ -47,5 +47,14 @@
         app:layout_constraintTop_toBottomOf="@+id/imageCaptureStepQuestionItself"
         app:srcCompat="@drawable/rsb_ic_circle_8dp" />
 
+    <ImageButton
+        android:id="@+id/imageCaptureImageStepQuestionAnswerFullscreen"
+        style="@style/rsb_image_capture_fullscreen_style"
+        app:layout_constraintEnd_toEndOf="@+id/imageCaptureImageStepQuestionAnswer"
+        android:contentDescription="Full Screen"
```

Comment:
Hardcoded string here. This should go to the multi-language spreadsheet and become a string.

Reply:
I was thinking when this MR is merged, I'll add this string to the Accessibility MR that's been blocked for a while now

Reply:
Ok.

Reply:
If you forget, you _pay the beers_